### PR TITLE
xdp: implement test case with different mtu

### PR DESCRIPTION
**Pull request by squirrelsc**
see details in commits.

### CODE REVIEW

File: lisa/tools/ip.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    @property
+    def command(self) -> str:
+        return "ip"
+
+    def _check_exists(self) -> bool:
+        return True
+
+    def restart_device(self, nic_name: str) -> None:
+        self.node.execute(
+            f"ip link set dev {nic_name} down;ip link set dev {nic_name} up", shell=True
```
Comment (dsrivastavv) [inline lisa/tools/ip.py:22]: Can we use `self.run` here instead of `self.node.execute`?

File: microsoft/testsuites/xdp/functional.py [hunk 2]
```diff
@@ -166,6 +166,47 @@ def verify_xdp_action_aborted(
...
+                remote_ip.set_mtu(remote_nic_name, mtu)
+
+                # tested mtu equals (ping mtu - IP headr (20) - ICMP header (8))
+                xdpdump.test(
+                    xdp_node_nic_name,
+                    remote_address=remote_address,
+                    ping_package_size=mtu - 28,
+                )
+        finally:
+            xdp_node_ip.set_mtu(xdp_node_nic_name, original_xdp_node_mtu)
```
Comment (dsrivastavv) [inline microsoft/testsuites/xdp/functional.py:207]: Should we mark the node to be recycled if we are unable to reset MTU back to its original value?

File: lisa/tools/ip.py (hunk 1 above)
Comment (squirrelsc) [reply lisa/tools/ip.py:22]: It has two commands, and joined with the `;`. The `run` makes it looks weird.

File: microsoft/testsuites/xdp/functional.py (hunk 2 above)
Comment (squirrelsc) [reply microsoft/testsuites/xdp/functional.py:207]: The dirty env is not implemented yet. This line is safe here, it's in `finally` block, will be run in any case. And if the set_mtu will fail, the mtu won't be changed in previous logic too.

**Review by dsrivastavv — APPROVED** [review]

**Pull request merged**